docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml

```diff
@@ -22,7 +22,7 @@
     <repositories>
       <repository>
         <id>jcenter</id>
```

Comment:
Same as above

Reply:
I've made the changes and pushed in new commit. Thanks!